bouture.js

```diff
@@ -12,16 +12,15 @@ tagNames.forEach(tagName => {
 
       Object.defineProperty(branche, 'getElement', {
```

Comment:
I had missed that in an earlier review, but `Object.defineProperty` is not needed. `branch.getElement = function(){...}` would work